lib/themes.dart

```diff
@@ -6,23 +6,30 @@ final ThemeData appTheme = ThemeData(
   fontFamily: 'Neuzeit',
   textSelectionTheme: TextSelectionThemeData(cursorColor: Colors.white),
   textTheme: TextTheme(
-      headline5: TextStyle(
-          color: Colors.white,
-          fontFamily: 'Neuzeit-Heavy',
-          fontWeight: FontWeight.w900)),
+    headline4: TextStyle(
+        color: Colors.white,
+        fontFamily: 'Neuzeit-Heavy',
```

Comment:
Hmm, I think we should use a constant for this font family name. It's repeated sometimes above 🤔

Reply:
it reads directly from the Font assets 🤔 , not a path or anything, so I'm not sure if it's worthy to create a const, because if you change the font file, you will have to change the entire code base too even with const (including const name) 🤔

Reply:
@markgravity could you please follow up the comment here? 😓